app/src/main/java/org/wikipedia/crash/CrashReportFragment.kt

```diff
@@ -0,0 +1,35 @@
+package org.wikipedia.crash
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import org.wikipedia.R
+
+class CrashReportFragment : Fragment() {
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
+        super.onCreateView(inflater, container, savedInstanceState)
+        val view = inflater.inflate(R.layout.fragment_crash_report, container, false)
+
+        view.findViewById<View>(R.id.crash_report_start_over).setOnClickListener {
```

Comment:
Note: this is a rare case when it's actually simpler to use `findViewById` rather than ViewBindings, because the XML for the crash report fragment uses an `include` which includes a chunk of XML that uses a `merge` node. This would make it a bit more complex to use as a ViewBinding scheme.

Reply:
Makes sense to me.